qiskit/ignis/experiments/calibration/methods/basic_1q.py

```diff
@@ -42,14 +42,11 @@ def __init__(self,
                  amplitude: float = 0.05,
                  sigma: float = 360,
                  duration: int = 1440,
-                 analysis: Optional[cal_base_fitter.BaseCalibrationAnalysis] = None,
+                 analysis: Optional[Calibration1DAnalysis] = None,
```

Comment:
```suggestion
                 analysis: Optional[Analysis] = None,
```

Reply:
Here we might want to keep `Calibration1DAnalysis`. The reason is that Spectroscopy, at least how it is setup now, will only ever scan one parameter. It therefor makes sense to restrict the type of analysis that is done.